content/en/team.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Team
+ 
+sidebar: false
+
+---
+
+*TO DO: Add a Google Maps snapshot illustrating where members of the NumPy team are from.*
+
+
+We are an international team on a mission to support scientific and research communities worldwide by building quality, open-source software. [**Join us**](https://numpy.org/contribute.md)!
```

Comment:
This link won't work, it should be `[**Join us**](/contribute)`.

With Nikola, starting with `/` will create a link relative to the root of the domain, while leaving off the `/` means relative to the current page.

Reply:
pushed a commit with a fix